linalg_ops/CMakeLists.txt

```
@@ -117,6 +117,26 @@ iree_cc_binary(
     iree::vm::cc
 )
 
+iree_cc_binary(
+  NAME
+    iree-e2e-conv2d-test
+  SRCS
+    "iree-e2e-conv2d-test.cc"
+  DEPS
+    ::e2e_test_util
```

Wrong dep name here
```suggestion
    ::test_utils
```
https://github.com/iree-org/iree-test-suites/actions/runs/10794339016/job/29938390318?pr=24#step:7:291

```
CMake Error at build/_deps/iree-src/build_tools/cmake/iree_cc_binary.cmake:147 (target_link_libraries):
  Target "iree-test-suites_iree-e2e-conv2d-test" links to:

    iree-test-suites::e2e_test_util
```